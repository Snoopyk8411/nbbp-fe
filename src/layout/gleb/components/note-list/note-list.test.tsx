import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-redux';
import NoteList from 'layout/gleb/components/note-list/Note-list';

let mockStore: any;
const mockStoreConf = configureStore();

const mockData = {};

describe('NoteList Component', () => {
  let props: { note: string };
  beforeEach(() => {
    mockStore = mockStoreConf(mockData);
  });

  it('NoteList snapshot', () => {
    const component = renderer.create(
      <Provider store={mockStore}>
        <NoteList />
      </Provider>,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
