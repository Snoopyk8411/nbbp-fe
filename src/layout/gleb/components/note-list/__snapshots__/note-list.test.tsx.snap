// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoteList Component NoteList snapshot 1`] = `
<div
  className="notes"
>
  <div>
    <h1
      className="title"
    >
      Notes
    </h1>
    <ul />
  </div>
  <div
    className="create_note"
  >
    <div
      className="create_note_form"
    >
      <textarea
        className="input"
        data-testid="AddNoteInputField"
        onChange={[Function]}
        onKeyPress={[Function]}
        placeholder="Write new note"
        rows={3}
        value=""
      />
      <button
        className="add_note_button"
        data-testid="AddNoteButton"
        disabled={false}
        onClick={[Function]}
        type="button"
      >
        Add
      </button>
    </div>
    <div />
  </div>
</div>
`;
